docs/VerifPlans/Simulation/privileged_spec/CV32E40X_Exceptions.json

```diff
@@ -0,0 +1,278 @@
+[
```

Comment:
> Removals are due to the test scenarios being impossible to meet.

![bilde](https://github.com/openhwgroup/cv32e40x-dv/assets/75431602/4dfcc2f3-97c8-4935-b1e9-caa61f5e00e1)

Reply:
There are no removals.

Reply:
Oh, right... the csv/json files didn't exist before this PR.

I'll wait for https://github.com/openhwgroup/cv32e40x-dv/pull/52 to be merge before looking at the diff again.

Reply:
There are still no removals, and the diff looks more like "new vplan items added" than it looks like "updated link2cov".
The diff doesn't seem to match the PR description.

@silabs-mateilga Do you have any idea if this is just a quirk with the json or the order of merging PRs or what it could be?

Reply:
Possible solution:
Checkout this branch, regenerate csv/json, see if diff looks ok (no conflict with the same vplan added in a separate PR etc).

Reply:
The last commit at least looks more like a diff than a new file.

Reply:
The PR's diff look mostly unchanged, but maybe that comes from this:

![bilde](https://github.com/openhwgroup/cv32e40x-dv/assets/75431602/ac6fe5a0-e3f8-41d9-ab24-808e35f8fe3e)

Reply:
I've resolved the merge conflict.